README.md

```diff
@@ -16,11 +16,11 @@ npm install react-entanglement
 
 ### Key concepts
 
-React Entanglement works by taking a component name and props in one end and **scattering** them over a **adapter** into a target where the **materialization** is done.
+React Entanglement works by taking a component's name and props and **scattering** them over an **adapter** into a target where the **materialization** is done.
 
 ### Example with simple API
 
-In some part of the application we prepare the props and select what element to render. Note that in this part (where the `Scatter` is used) we don't need a reference to the component that we intend to render, since it's sent via the adapter by name.
+In some part of the application we prepare the props and select what element to render. Note that in this snippet–where the `Scatter` is used–we don't need a reference to the component that we intend to render, since it's sent via the adapter by name.
```

Comment:
Oh, it's actually an _em_ dash. Take a look in the rendered file, it's not a dash.

I can put the parens back if you like it better though.